sartools/macros.xml

```diff
@@ -1,11 +1,10 @@
 <macros>
 
-    <token name="@WRAPPER_VERSION@">1.6.3</token>
+    <token name="@TOOL_VERSION@">1.7.3</token>
 
     <xml name="requirements">
         <requirements>
-            <requirement type="package" version="1.6.3">r-sartools</requirement>
-            <requirement type="package" version="1.6.0">r-optparse</requirement>
+            <requirement type="package" version="1.7.3">r-sartools</requirement>
```

Comment:
You can use the `@TOOL_VERSION@` token here

Reply:
done